### PR TITLE
virtual files: introduce branding option for showing ui option

### PR DESCRIPTION
**Pull request by ckamm**

### CODE REVIEW

**Comment by ckamm** [comment]
Actually I shouldn't have removed it, it should have become a branding option

**Comment by ckamm** [comment]
How to make the testcloud client show the option too?

**Comment by guruz** [comment]
> How to make the testcloud client show the option too?

That's in an internal repo, I can do that part..

File: src/libsync/theme.cpp [hunk 1]
```diff
@@ -555,4 +555,9 @@ QString Theme::versionSwitchOutput() const
     return helpText;
 }
 
+bool Theme::showVirtualFilesOption() const
+{
+    return false;
```
Comment (ogoffart) [inline src/libsync/theme.cpp]: Maybe  `return ConfigFile().showExperimentalOptions();`  ?

Comment (ckamm) [reply src/libsync/theme.cpp]: good point, done. makes Theme depend on ConfigFile though

**Pull request merged**